doc/userguide/devguide/contributing/backports-guide.rst

```diff
@@ -94,7 +100,7 @@ backporting process once the PR for main has been merged. Then:
       backported.
     * *Bring each commit into the new branch,* one at a time -- starting with the
       oldest commit. Use ``git cherry-pick -x commit-hash``, where ``commit-hash``
-      is the hash to the commit already in main or main-7.0x that is being
+      is the hash to the commit already in main or main-8.0x that is being
```

Comment:
Q: do the cherry-pick commit checks check both the trees?

Reply:
yes